packages/bun-types/sqlite.d.ts

```diff
@@ -82,6 +82,40 @@ declare module "bun:sqlite" {
              * Equivalent to {@link constants.SQLITE_OPEN_READWRITE}
              */
             readwrite?: boolean;
+
+            /**
+             * When set to `true`, integers are returned as `bigint` types.
+             *
+             * When set to `false`, integers are returned as `number` types and truncated to 52 bits.
+             *
+             * @default false
+             * @since v1.1.14
+             */
+            safeInteger?: boolean;
```

Comment:
Name is confusing, just name it `bigint: boolean`

Reply:
that's what i had originally, but this is consistent with the rest. It also implies that bigint is not supported - which is not true.